channel2/message_test.go

```diff
@@ -61,3 +61,126 @@ func Test_getRetryVersionFor(t *testing.T) {
 		})
 	}
 }
```

Comment:
Can you add a test and see if decode still works on values encoded with binary.LittleEndian?

Reply:
added as part of Uint64 tests